src/MCPClient/lib/clientScripts/removeUnneededFiles.py

```diff
@@ -2,21 +2,32 @@
 
 import os
 import sys
+import shutil
 
 # databaseFunctions requires Django to be set up
 import django
 django.setup()
 # archivematicaCommon
 from custom_handlers import get_script_logger
 from databaseFunctions import fileWasRemoved
+import ConfigParser
```

Comment:
Nitpick: Since this is part of python's standard library, can you move it to the line above `import os`?